presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestReorderJoins.java

```diff
@@ -56,8 +56,8 @@
 {
     private RuleTester tester;
 
-    // TWO_ROWS are used to prevent node from being scalar
-    private static final ImmutableList<List<Expression>> TWO_ROWS = ImmutableList.of(ImmutableList.of(), ImmutableList.of());
+    // TWO_LEGACY_ROWS, TWO_ROWS are used to prevent node from being scalar
```

Comment:
Did you plan to change the variable name or something?

Reply:
forgot to remove during refactoring..